Ready for review

BrittanyBlake opened the pull request:
## Building with Responsive Design

-Added media queries with 3 breakpoints. 
-Added navigation bar with different layouts for each breakpoint.
-Added different content sections with different layouts for each breakpoint. 
-Added footer section with icons. 

[Live Version](https://raw.githack.com/Buyaki01/the-next-web-clone/feature/index.html)

Buyaki01 committed:
Add index.html, stickler and stylelint files

Buyaki01 committed:
Add empty html semantics

Buyaki01 committed:
Add size and background colors to main elements layout

Buyaki01 committed:
Add nav1

Buyaki01 committed:
add drop-down menu

Buyaki01 committed:
Add navbar2

Buyaki01 committed:
Add navbar3

BrittanyBlake committed:
Add article images to main

BrittanyBlake committed:
Style article images

BrittanyBlake committed:
Add advertisement section

BrittanyBlake committed:
Add media query for article section

Buyaki01 committed:
Fixed the search icon and removed the drop down menu

Buyaki01 committed:
Add media queries for the first nav-bar

BrittanyBlake committed:
Add image folder

Buyaki01 committed:
Add media query for nav-bar2

Buyaki01 committed:
Add changes to media query

BrittanyBlake committed:
add div to hidden nav

BrittanyBlake committed:
add media query to nav 2

BrittanyBlake committed:
add media queries nav bar

BrittanyBlake committed:
finish navbar media quieries

BrittanyBlake committed:
Add images

BrittanyBlake committed:
Add latest news section

BrittanyBlake committed:
Add grid to latest news section

BrittanyBlake committed:
update latest news section

BrittanyBlake committed:
Add first media query to latest news section

BrittanyBlake committed:
Add images

BrittanyBlake committed:
Style latest articles in media query

BrittanyBlake committed:
add latest funding section

BrittanyBlake committed:
add strong tag to text in latest news section

BrittanyBlake committed:
Add second media query to latest articles section

BrittanyBlake committed:
Add image

BrittanyBlake committed:
Add another item to latest funding section

BrittanyBlake committed:
style latest funding items

BrittanyBlake committed:
Format document

BrittanyBlake committed:
modify latest funding section div tags

BrittanyBlake committed:
add grid for latest funding section

BrittanyBlake committed:
add to latest funding section

BrittanyBlake committed:
Finish media queries for latest funding section

Buyaki01 committed:
Added content to the latest-section

Buyaki01 committed:
Added the images

Buyaki01 committed:
Add style to latest-section

BrittanyBlake committed:
add closing tag to div

Buyaki01 committed:
Add grid

BrittanyBlake committed:
modify class names

BrittanyBlake committed:
modify grid

Buyaki01 committed:
Add content to the grid columns and rows in the latest-section

Buyaki01 committed:
Modify grid

BrittanyBlake committed:
Add gradient image

BrittanyBlake committed:
add images

BrittanyBlake committed:
add text to image

Buyaki01 committed:
Add to styles.css

Buyaki01 committed:
Made changes to the latest-section

BrittanyBlake committed:
Add logo and text to top grid images

BrittanyBlake committed:
Add background images and linear gradients to top grid section

BrittanyBlake committed:
deleted image

BrittanyBlake committed:
Fix media queries for first articles section

BrittanyBlake committed:
modify media queries for nav

BrittanyBlake committed:
change article images

BrittanyBlake committed:
add hover properties and style text

Buyaki01 committed:
Add grid to the media query

BrittanyBlake committed:
format document

BrittanyBlake committed:
modify grid

Buyaki01 committed:
Add grid to the second media query

BrittanyBlake committed:
modify image sizing

Buyaki01 committed:
Add latest deals section and images

Buyaki01 committed:
Add styling to texts in the latest-deals

Buyaki01 committed:
Add grid items in the latest-deals section

Buyaki01 committed:
Add content to the README file

BrittanyBlake committed:
modify ad

BrittanyBlake committed:
Add media query to latest deals

BrittanyBlake committed:
modify nav bar

BrittanyBlake committed:
add media query to latest deals section

BrittanyBlake committed:
Add footer

BrittanyBlake committed:
Style footer

BrittanyBlake committed:
modify footer order

BrittanyBlake committed:
add layout to footer

BrittanyBlake committed:
add screenshot and link to website to read me

BrittanyBlake committed:
update screenshot image

BrittanyBlake committed:
Add image to folder

BrittanyBlake committed:
remove live demo

BrittanyBlake committed:
add media queries to footer

BrittanyBlake committed:
Make navbar fixed

BrittanyBlake committed:
fix linter errors

BrittanyBlake committed:
fix html validator errors

BrittanyBlake committed:
fix linter errors

BrittanyBlake committed:
Fix linter errors

BrittanyBlake committed:
add gh-pages

rOluochKe reviewed (changes requested):
Hello @BrittanyBlake and @Buyaki01 

#### That was amazing job you put in :clap: However, there are a few changes you need to work on before your project could be submitted.
- Use the new HTML5 semantics like `section, article, aside, footer, main` to structure your code. An example 
![class](https://user-images.githubusercontent.com/8114528/74935856-0fc94700-53fa-11ea-90e4-ebe0a32d7b84.PNG)
- You can improve your code, instead of creating a div and giving it a class `articles`, you can create an article tag
- Some of your images are stretched on small devices, make the responsive
![strettched](https://user-images.githubusercontent.com/8114528/74936091-92520680-53fa-11ea-85a5-72a46b5010f1.PNG)

### Make these changes and resubmit for another code review. All the best.
- Happy coding!

BrittanyBlake committed:
Add semantic html tags

BrittanyBlake committed:
update image sizes

BrittanyBlake commented:
## Building with Responsive Design

-Added semantic HTML tags throughout code. 
-Adjusted image sizes so they are no longer stretched on small screens.

[Live Version](https://raw.githack.com/Buyaki01/the-next-web-clone/feature/index.html)

davidmrtz-dev commented:
### **Status: Approved.** ✅ 
### Hi Team. 😄 
### Very good job so far, your project meets all the requirements. 💯 👍 

Let's merge this branch and move onto the next project. 🆒 ⏭ 

****

Happy coding! 😄 💻 

**[TSE:  David Elí]** 👨‍💻 
Github: [@davitomix](https://github.com/davitomix)
Twitter: [@davitomix](https://twitter.com/davitomix007)
Linkedin: [David Elí Martínez García](https://www.linkedin.com/in/davideligarcia/)
